daml-script/dump/src/main/scala/com/daml/script/dump/TreeUtils.scala

```diff
@@ -33,6 +50,14 @@ object TreeUtils {
     }
   }
 
+  def partiesInContracts(contracts: Iterable[CreatedEvent]): Set[String] = {
+    var parties: Set[String] = Set()
+    contracts.foreach { value =>
```

Comment:
looks like a `foldMap` :slightly_smiling_face:

Reply:
Yup, this was a copy paste from `partiesInTree`. I've changed it a `foldMap`.